run-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+
+mvn test -B
+
+# Break CI build on PRs where committed differ from generated sources
+if [ -n "$(git status -su javaslang/src-gen)" ]; then
+  exit 1
+fi
```

Comment:
Please move this file to the folder `.travis/`. There we collect all necessary CI build scripts.